src/alpaka/test/alpaka/HistoContainer_t.cc

```diff
@@ -8,10 +8,10 @@
 #include "AlpakaCore/alpakaWorkDivHelper.h"
 #include "AlpakaCore/HistoContainer.h"
 
-using namespace ALPAKA_ACCELERATOR_NAMESPACE;
-
```

Comment:
Sorry, but actually here (and in other unit test files) the `using namespace ...` is ok. It's just in headers where it causes problems (although it does make the namespace of the various types more clear in source files as well).

Reply:
Yes but at this stage, found it more consistent to remove it everywhere, rather than having a mix of 'using namespace' and ALPAKA_ACCELERATOR_NAMESPACE::